src/components/index/community.tsx

```diff
@@ -1,79 +1,140 @@
-                }
-            />
-        </div> 
-    );
-}
+  const pageRedirect = (url: string) => {
+    window.open(url, "_blank");
+  };
+
+  const communityLinks = links.Body["Community"];
+
+  return (
+    <div className="community-section">
+      <Section
+        sectionTitle="Community"
+        sectionDescript="Join NetLogo community and start contributing today."
+        sectionGap={1.88}
+        sectionPaddingBot={3.75}
+        backgroundColor="#F2F2F5"
+        borderRadius={0}
+        moreButton={false}
+        body={
+          //   <div className="community-content">
+          //         <div className="preview-content">
+          //             <div className="preview-display">
+          //                 {previewImage ? <img className="community-preview-image" src={previewImage} alt="Preview" /> : 'No preview available'}
+          //             </div>
+          //             <Button
+          //                 colorClass="blue-button"
+          //                 padding="1rem 3rem"
+          //                 fontSize="1.125rem"
+          //                 text="JOIN DISCOURSE"
+          //                 style={{ alignSelf: 'stretch' }} // make it grow to the full size
+          //             />
+          //         </div>
+          //         <div className="community-models">
+          //             {communityPosts.map((post: CommunityPost, index: number) => (
+          //                 <div
+          //                     className="community-post-cont"
+          //                     key={index}
+          //                     onClick={() => setPreviewImage(post.image)}
+          //                 >
+          //                     <div className="community-post-icon">
+          //                         <img src={communityIcon.src} className="community-post-icon-image"/>
+          //                     </div>
+          //                     <div className="community-post-descript">
+          //                         <div className="community-post-user-date-cont">
+          //                             <span> {post.author} </span>
+          //                             <span> {post.date} </span>
+          //                         </div>
+          //                         <span className="community-post-title"> {"New model: "}
+          //                             <a href={post.link} className="community-post-link">{post.project}</a>
+          //                         </span> </div>
+          //                 </div>
+          //             ))}
+          //         </div>
+          //     </div>
+          <div>
+            <div
+              style={{
+                display: "flex",
+                flexWrap: "wrap",
+                justifyContent: "center",
+                gap: "2%",
+              }}
+              className="get-netlogo-content"
+            >
+              <div className="get-item" style={{backgroundColor:"white"}}>
```

Comment:
Make a component for this type of card and then include that component once for each card. It's more DRY and  will make it easier to switch to Directus later (where we will pull data and loop through it and make a card for each one)